web/src/main/resources/templates/contact.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+    <head th:replace="fragments/head"></head>
+
+    <body class="bg-dark bg-opacity-10">
+        <div th:replace="fragments/navigation :: top-navigation"></div>
+        <div class="container py-5">
+            <div class="row h-100 align-items-center py-5 form container">
+                <div class="col-lg-6">
+                    <h1 class="display-4 header-style center">Contact Us</h1>
+                    <p class="center"><br>Contact the team for help with any enquiries. Fill in the enquiry form or use our details below to get in touch.</p>
```

Comment:
```suggestion
                    <p class="center"><br>Contact the team for help with any enquiries. </p>
```
No Contact Form included.